apps/feedahelp/src/styles/imageCarousal.css

```diff
@@ -24,31 +24,31 @@
   animation: slide var(--d) infinite, z-order var(--d) infinite steps(1);
 }
 
-.gallery img:last-child {
+.gallery .img:last-child {
   animation-name: slide, z-order-last;
 }
 
-.gallery > img:nth-child(1) {
+.gallery > .img:nth-child(1) {
   animation-delay: calc(0*var(--d));
   --r: -2deg;
 }
 
-.gallery > img:nth-child(2) {
+.gallery > .img:nth-child(2) {
   animation-delay: calc(-0.2*var(--d));
   --r: -6deg;
 }
 
-.gallery > img:nth-child(3) {
+.gallery > .img:nth-child(3) {
   animation-delay: calc(-0.4*var(--d));
   --r: 9deg;
 }
 
-.gallery > img:nth-child(4) {
+.gallery > .img:nth-child(4) {
   animation-delay: calc(-0.6*var(--d));
   --r: 10deg;
 }
 
-.gallery > img:nth-child(5) {
+.gallery > .img:nth-child(5) {
```

Comment:
is this working, becouse looks fishy?
you chanded `img` whihc where css looks for element to now `.img` which is s css class?

Ding dong bell pussy is in the well

Reply:
look at the others